### PR TITLE
Middled the Pagination

### PR DESCRIPTION
**Pull request by CodeSarthak**
Middled the pagination after converting HTML to JADE file.
paginationscreenshot

### CODE REVIEW

**Comment by CodeSarthak** [comment]
This resolves : https://github.com/publiclab/plots2/issues/3732

**Comment by DivyeshPuri** [comment]
This is not the correct way of centering the element as it will make the pagination irresponsive. The correct way of doing is adding ` display: flex; justify-content: center`.

**Review by DivyeshPuri — CHANGES_REQUESTED** [review]
This is not the correct way of centering the element as it will make the pagination irresponsive. The correct way of doing is adding `display: flex; justify-content: center`.

File: views/show.jade [hunk 1]
```diff
@@ -54,6 +54,10 @@ block content
       button#raw(type="button" class="btn active") Raw
       button#ndvi(type="button" class="btn") NDVI
       button#nir(type="button" class="btn") NIR
+       style.
+      .btn-group {
+      padding-left : 400px;
```
Comment (adisapphire) [inline views/show.jade]: @CodeSarthak this will increase padding only from left but we need it in center of page.
Thanks

**Review by adisapphire — COMMENTED** [review]
Please make suggested changes.
Thanks

**Comment by avsingh999** [comment]
@CodeSarthak  you must check this also in mobile view in your browser
thanks :)

File: views/show.jade (hunk 1 above)
Comment (CodeSarthak) [reply views/show.jade]: Yeah, sure. I got it, will make the necessary changes.

**Comment by CodeSarthak** [comment]
Changes done.

**Comment by shivani-kapania** [comment]
@CodeSarthak can you please share a gif or screenshots?

**Comment by CodeSarthak** [comment]
paginationscreenshot
PFA the screenshot.

Thanks and Regards.

**Review by hackta — APPROVED** [review]
Looks good to me. Great Work 🎉

**Comment by adisapphire** [comment]
looks awesome!
Thanks,

**Comment by igniteeng000** [comment]
Please provide a screenshot for mobile view also. Great work, Thanks.

**Comment by CodeSarthak** [comment]
Thanks for the help and support. Here is the screenshot for mobile view : 
mobileview

**Review by DivyeshPuri — APPROVED** [review]
Great Work 👍

**Review by avsingh999 — APPROVED** [review]

**Comment by avsingh999** [comment]
@jywarren it looks good to me, and I thinks it's ready to 
thanks :)

**Comment by jywarren** [comment]
Super! Sorry it took me a couple days to look over! Great work!!! 👍 👍 👍

**Pull request merged**